codejam-virtual-keyboard/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "dependencies": {
+    "element-resize-event": "^3.0.3",
+    "normalize.css": "^8.0.1"
+  },
+  "devDependencies": {
+    "css-loader": "^3.2.0",
+    "del": "^5.1.0",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-plugin-import": "^2.18.2",
+    "file-loader": "^4.2.0",
+    "glob": "^7.1.5",
+    "gulp": "^4.0.2",
+    "gulp-eslint": "^6.0.0",
+    "gulp-htmlhint": "^2.2.1",
+    "gulp-stylelint": "^10.0.0",
+    "html-loader": "^0.5.5",
+    "html-webpack-plugin": "^3.2.0",
+    "mini-css-extract-plugin": "^0.8.0",
+    "node-pretty-log": "^1.1.0",
+    "require-dir": "^1.2.0",
+    "resolve-url-loader": "^3.1.0",
+    "sass": "^1.23.1",
+    "sass-loader": "^8.0.0",
+    "style-loader": "^1.0.0",
+    "stylelint": "^11.1.1",
+    "stylelint-config-htmlacademy": "^0.1.4",
+    "webpack": "^4.41.2",
+    "webpack-cli": "^3.3.9",
+    "webpack-dev-server": "^3.9.0"
+  },
+  "scripts": {
+    "build": "webpack",
+    "start": "webpack-dev-server --open",
+    "watch": "webpack --watch",
+    "clean-dist": "gulp clean",
+    "lint": "node node_modules/gulp/bin/gulp.js lint",
```

Comment:
There should be script for running `eslint`.
The script for running css linter could be named like `css-lint`.